spec/interactors/users/generate_access_token_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe Users::GenerateAccessToken do
+  describe ".call" do
+    user = User.create(
+      email: "user@admin.ru",
+      password: "123456"
+    )
+
+    let!(:interactor) { described_class.new(user: user) }
+
+    it "existince of token" do
+      interactor.run
+
+      expect(interactor.context.access_token).present?
```

Comment:
Does it work? You should write there smh like:
```suggestion
      expect(interactor.context.access_token).to be_present
```

Reply:
please explain to me what is the difference?

Reply:
https://github.com/rspec/rspec-expectations#readme